worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -1102,7 +1102,8 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
         throw new IOException(s"No available disks! suggested mountPoint $suggestedMountPoint")
       }
 
-      if (dirs.isEmpty && location.getStorageInfo.HDFSAvailable()) {
+      if ((dirs.isEmpty && location.getStorageInfo.HDFSAvailable())
```

Comment:
Why not use `(dirs.isEmpty || hasHDFSStorage) && location.getStorageInfo.HDFSAvailable()`? BTW, Is it inconsistent between `hasHDFSStorage` and `HDFSAvailable`?